invoice.go

```diff
@@ -29,6 +29,47 @@ const (
 	InvoiceBillingReasonUpcoming              InvoiceBillingReason = "upcoming"
 )
 
+// InvoicePaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage is the
+// preferred language of the Bancontact authorization page that the customer is
+// redirected to.
+type InvoicePaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage string
+
+// List of values that InvoicePaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage can take.
```

Comment:
Not sure about this -- the general rule is "define enums for resources, but not for params" but we don't seem to be explicit about "language" enums anywhere else.

Reply:
Hmm, IMO we should probably just make it a string because (1) it's documented as string as opposed to enum in the API ref, (2) this could potentially have a lot of different values, (3) if the product team added a new value without the library I'm not sure that we'd be able to handle it that well, and (4) it looks like it's a string on payment intent, setup attempt, and charge.

Thoughts?